packages/wasm-transformer/src/unoptimized.ts

```diff
@@ -4,10 +4,17 @@ import wasmTransformerWasmUrl from "../wasm-pack/web/wasm_transformer_bg.wasm";
 // @ts-ignore
 import * as wasmTransformerWasmPack from "../wasm-pack/web/wasm_transformer";
 
+import { bigInt } from "wasm-feature-detect";
+
 // @ts-ignore
 const initPromise = wasmTransformerWasmPack.default(wasmTransformerWasmUrl);
 
 export const lowerI64Imports = async (wasmBinary: Uint8Array) => {
+  let isBigIntSupported = bigInt();
```

Comment:
This should be `await bigInt()`.
Why tests are passing?

Reply:
Fixed this! 

Tests pass as they only check that the transformation doesn't crash, and the actual transformation logic is tested on the rust side. So all in all, I think this is a good catch on your part 😄 👍 Thank you!